test/hotspot/jtreg/compiler/lib/ir_framework/TestFramework.java

```diff
@@ -593,27 +593,44 @@ private void installWhiteBox() {
      * Disable IR verification completely in certain cases.
      */
     private void disableIRVerificationIfNotFeasible() {
-        if (irVerificationPossible) {
-            irVerificationPossible = Platform.isDebugBuild() && !Platform.isInt() && !Platform.isComp();
-            if (!irVerificationPossible) {
-                System.out.println("IR verification disabled due to not running a debug build (required for PrintIdeal" +
-                                   "and PrintOptoAssembly), running with -Xint, or -Xcomp (use warm-up of 0 instead)");
-                return;
-            }
+        if (!irVerificationPossible) {
+            return;
+        }
 
-            irVerificationPossible = hasIRAnnotations();
-            if (!irVerificationPossible) {
-                System.out.println("IR verification disabled due to test " + testClass + " not specifying any @IR annotations");
-                return;
-            }
+        boolean debugTest, intTest, compTest, irTest, nonWhiteListedTest;
 
-            // No IR verification is done if additional non-whitelisted JTreg VM or Javaoptions flag is specified.
-            irVerificationPossible = onlyWhitelistedJTregVMAndJavaOptsFlags();
-            if (!irVerificationPossible) {
-                System.out.println("IR verification disabled due to using non-whitelisted JTreg VM or Javaoptions flag(s)."
-                                   + System.lineSeparator());
-            }
+        debugTest = Platform.isDebugBuild();
+        intTest = !Platform.isInt();
+        compTest = !Platform.isComp();
+        irTest = hasIRAnnotations();
+        // No IR verification is done if additional non-whitelisted JTreg VM or Javaoptions flag is specified.
+        List<String> nonWhiteListedFlags = anyNonWhitelistedJTregVMAndJavaOptsFlags();
+        nonWhiteListedTest = nonWhiteListedFlags.isEmpty();
```

Comment:
You can directly add the type declarations here:
```
boolean debugTest = Platform.isDebugBuild();
boolean intTest = !Platform.isInt();
boolean compTest = !Platform.isComp();
boolean irTest = hasIRAnnotations();

boolean nonWhiteListedTest = nonWhiteListedFlags.isEmpty();

Reply:
Yes, fixed.